server/gitrest/package.json

```diff
@@ -21,27 +21,35 @@
     "tsc": "tsc"
   },
   "dependencies": {
-    "@fluidframework/gitresources": "^0.1012.0",
+    "@fluidframework/gitresources": "^0.1014.0-7764",
+    "@fluidframework/protocol-base": "^0.1014.0-7764",
+    "@fluidframework/protocol-definitions": "^0.1014.0-7764",
+    "@fluidframework/server-services-client": "^0.1014.0-7764",
+    "@fluidframework/server-services-core": "0.1014.0-7764",
```

Comment:
Do you need all of these packages?

Reply:
added them at first to explicitly set the version of services-core's dependencies while testing, but don't need some of the dependencies. Removed.